collect_app/src/main/res/values/strings.xml

```diff
@@ -715,4 +715,5 @@
     <string name="qr_code_not_found">QR Code not found in the selected image</string>
     <string name="not_granted_permission">The activity you are trying to start requires a permission which is not granted.</string>
     <string name="basemap_source_unavailable">Sorry, %s basemaps are not available on this device.  Please choose another basemap in General Settings &gt; Maps.</string>
+    <string name="exception_while_reading_csv">Exception thrown while reading csv file</string>
```

Comment:
I'd tend to leave this untranslated because the issue is (I believe) uncommon and the message is not very informative. If it's going to be translated, I think it should at least help the user figure out what went wrong.

Perhaps using the localized exception message would be a good option? I'm not sure why we don't do that very often. Maybe because a lot of systems don't actually localize the exception text?

Reply:
We could just change the text to inform the user that there are unsupported characters in their CSV?

Reply:
That's not the only way to get an `SQLiteException`, right?

Reply:
Ah true. If there is a way to single that out and report it in a more "human" way though it feels useful. It's a problem the user could potentially fix.

Reply:
Sorry I'm not active here but I just don't have any strong preferences. Since it's a rare case that may be caused by different bugs, maybe we should display an error message (`e.getMessage`) without any translations and that's all?

Reply:
Yeah I'm happy with that as a start! 

It seems a lot of our errors are fairly "technical" in their wording so I'm just interested in ways to improve that. That's probably a larger effort down the road though.